source/docs/Titan/programing-the-titan.rst

```diff
@@ -72,6 +86,28 @@ Setting Motor Speed
             {
                 motor.Set(speed);
             }
+            
+    .. tab:: Roscpp
+    
+         .. code-block:: c++
+            :linenos:
+            
+            /**
+             * Sets the speed of a motor
+             * 
+             * @param speed range -1.0 to 1.0 (0 stop)
+             */
+             
```

Comment:
you have a comment block for a function but there is no function?

Reply:
It's not meant to be a function, I'll update this